.golangci.yml

```diff
@@ -74,6 +73,8 @@ linters-settings:
         disabled: true
       - name: nested-structs
         disabled: true
+  gofmt:
+    simplify: false
```

Comment:
Is there a reason why we are changing this here?

Reply:
actually, I just forgot to remove it